lib/pbio/drv/gpio/gpio_ev3.c

```diff
@@ -32,7 +32,7 @@ static void pbdrv_gpio_alt_gpio(const pbdrv_gpio_t *gpio) {
 }
 
 static void gpio_write(const pbdrv_gpio_t *gpio, uint8_t value) {
-    if (!gpio) {
+    if (!gpio || !gpio->bank) {
```

Comment:
Technically, there is a GPIO pin for `pbdrv_ioport_platform_data_vcc_pin` - it is just shared with the ADC and the USB 1.1 host port. So I would be inclined to add some compile-time options to show that EV3 is different in this regard rather than than having a non-value assigned to a GPIO.